docs/deploy_narrative.md

```diff
@@ -1,64 +1,109 @@
 ## Narrative Development & Deployment HowTo
 
+This document describes how to release and deploy the Narrative Interface app onto the different KBase environments. It's intended for KBase developers and operators.
 
-### Overview
+- Last modified: Sep. 22, 2020
 
-1. Create a "development" image for CI.
-1. Deploy image to CI.
-1. Once tested in CI, create a single "production" image for appdev, next, & prod.
-1. Tag the "production" image for deployment using the `deploy_tag.sh` script.
-1. Deploy the "production" image in the relevant environments.
+**_Table of Contents_**
 
-### CI
+-   [Deployment flow overview](#deployment-flow-overview)
+-   [Deploying CI](#deploying-ci)
+    -   [Create CI Image](#create-ci-image)
+    -   [Release CI](#release-ci)
+-   [Stage a Final Release](#stage-a-final-release)
+-   [Deploying Next, Appdev, and Prod](#deploying-next,-appdev-&-prod)
+    -   [Create Release Image](#create-release-image)
+    -   [Tag & Deploy Image](#tag-&-deploy-image)
+-   [Image URLs](#image-urls)
+
+### Deployment Flow Overview
+
+1.  Create a "development" image for CI.
+2.  Deploy image to CI.
+3.  Once tested in CI, draft a release by bumping the version and creating a Github release tag.
+4.  Create a single production image for appdev, next, & prod.
+5.  Tag the production image for deployment using the `deploy_tag.sh` script.
+6.  Deploy the production image in the relevant environments.
+
+### Deploying CI
+
+The Continuous Integration environment - <https://ci.kbase.us> - is intended for testing and demoing of new features that may not always work right. It's also intended to be continuously upgraded and redeployed, so this process is designed to be as easy as possible.
 
 #### Create CI Image
 
-1. Create a new temporary branch from the `develop` branch.
-1. Add a _single_ bug fix or new feature to your temporary branch.
-1. Create a new [pull request](https://github.com/kbase/narrative/compare) to merge your branch into `develop`.
-1. Ensure the automated [build test](https://github.com/kbase/narrative/actions?query=workflow%3A%22Build+Test+Image%22) completes successfully.
-1. Complete review and testing.
-1. Merge pull request.
+1.  Create a new temporary branch from the `develop` branch.
+2.  Add a _single_ bug fix or new feature to your temporary branch.
+3.  Create a new [pull request](https://github.com/kbase/narrative/compare) to merge your branch into `develop`.
+4.  Ensure the automated [build test](https://github.com/kbase/narrative/actions?query=workflow%3A%22Build+Test+Image%22) completes successfully.
+5.  Complete review and testing.
+6.  Merge pull request.
 
-#### Deploy CI
+#### Release CI
 
 Once a pull request is merged from a temporary branch to `develop` a new _development_ image will be available at:
 `docker.pkg.github.com/kbase/narrative/narrative-develop:latest`
 
 Simply restart the `narrative` container for CI in Rancher. No other action is needed.
```

Comment:
Should there be instructions on how to restart the narrative container in Rancher?

Reply:
I'm touching base with some of the devops crew to get that updated. I think we should have instructions for doing CI in here, as it's mostly meant to be released at-will by developers, but the other environments need operator intervention.

Reply:
Ok, worked with @jsfillman to verify this and reworked the text a touch.